rudra/deployments/emr_scripts/emr_config.py

```diff
@@ -15,7 +15,6 @@ def __init__(self, name, log_uri, ecosystem, s3_bootstrap_uri, training_repo_url
         self.instances = {
             'KeepJobFlowAliveWhenNoSteps': False,
             'TerminationProtected': False,
-            'Ec2KeyName': 'Zeppelin2Spark',
```

Comment:
@abs51295  Assuming it should work without this hopefully?

Reply:
Yes, because it isn't a required parameter. https://docs.aws.amazon.com/emr/latest/APIReference/API_Ec2InstanceAttributes.html

Reply:
Sure sounds good